addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -115,6 +115,14 @@ var SnippetEditor = Widget.extend({
             this.draggableComponent = this._initSmoothScrollOnDrag(".o_move_handle", this.$el);
             if (!this.$target[0].matches("section")) {
                 this.draggableComponentImgs = this._initSmoothScrollOnDrag("img", this.$target);
+            } else {
+                // Prevent OdooEditor drag and drop.
+                this.preventOdooEditorDragListener = this.$target[0].addEventListener("dragstart",
+                    (ev) => {
+                        ev.preventDefault();
+                        ev.stopPropagation();
+                    }
+                );
```

Comment:
- Do we also want to prevent the drag and drop of the text? Because with this change it is the case.. Otherwise it must be limited to images.... 

- Also why adding an event on each section? This still allows the `OdooEditor` drag and drop in other places such as the menu or the `o_footer_copyright` for example. Shouldn't there be only one event listener on the whole `wrapwrap` ? (on content editable and not editable). It is still possible to drag and drop a product image on the /shop page or a product page inside a section.

- Also I wonder if preventing "dragstart" (when we only want to prevent that of the editor) is not too much because there could be snippets in which we want to use a "dragstart" listener.

Reply:
I also spotted a bug when you drag and drop an image in a mega menu, there is an infinite loader in the options panel.

![Capture d’écran du 2023-09-06 10-07-05](https://github.com/odoo/odoo/assets/52911687/53143ba0-bc00-42d2-bdc3-9bf4a013f8e3)

Reply:
> I also spotted a bug when you drag and drop an image in a mega menu, there is an infinite loader in the options panel. (it is not related to your changes)

I created [a task](https://www.odoo.com/web#id=3495730&menu_id=4720&cids=1&action=333&active_id=974&model=project.task&view_type=form) to fix it :+1: